tests/dispatch_workqueue.c

```diff
@@ -0,0 +1,72 @@
+#include <bsdtests.h>
```

Comment:
Oh this looks like a very nice addition.  This was something that we didn't catch on the original windows port and had to figure it out much later.